src/util/apis/generated/models/index.d.ts

```diff
@@ -47108,8 +47090,7 @@ export interface GetOKResponseAzureSubscription {
   isMicrosoftInternal?: boolean;
 }
 
-export interface
-GetOKResponseModelModelModelModelModelModelModelModelModelModelModelModelModelModel {
+export interface GetOKResponseModelModelModelModelModelModelModelModelModelModelModelModelModelModel {
```

Comment:
recognizing that it hasn't been changed, but `GetOKResponseModelModelModelModelModelModelModelModelModelModelModelModelModelModel` is really interesting?

Reply:
I had to remove the newlines. But yeah the names are pretty hilarious.